src/Randomizer.SMZ3.Tracking/Tracker.cs

```diff
@@ -1788,12 +1793,12 @@ public void ClearDungeon(DungeonInfo dungeon, float? confidence = null)
             }
 
             Say(x => x.DungeonCleared, dungeon.Name);
-            if (inaccessibleLocations.Count > 0)
+            if (inaccessibleLocations.Count > 0 && (HintsEnabled || SpoilersEnabled))
             {
                 var anyMissedLocation = inaccessibleLocations.Random(s_random);
                 var locationInfo = WorldInfo.Location(anyMissedLocation);
                 var missingItemCombinations = Logic.GetMissingRequiredItems(anyMissedLocation, progress);
-                if (missingItemCombinations.Any() && (HintsEnabled || SpoilersEnabled))
```

Comment:
I'm not sure about these changes. Obviously, Tracker should not do this if this isn't actually where the item came from (in the case of cheats).

But requiring hints or spoilers to be turned on effectively removes these lines entirely. I would instead use the spoiler confidence threshold for this, to prevent potentially misheard commands from triggering this accidentally, but still have this trigger if the command was actually intentional.